x-pack/plugins/reporting/server/config/schema.ts

```diff
@@ -31,11 +30,11 @@ const KibanaServerSchema = schema.object({
 }); // default values are all dynamic in createConfig$
 
 const QueueSchema = schema.object({
-  indexInterval: schema.string({ defaultValue: 'week' }),
+  indexInterval: schema.string({ defaultValue: 'week' }), // no schema.duration: used to create weekly indices
   pollEnabled: schema.boolean({ defaultValue: true }),
   pollInterval: schema.number({ defaultValue: 3000 }),
   pollIntervalErrorMultiplier: schema.number({ defaultValue: 10 }),
-  timeout: schema.number({ defaultValue: moment.duration(2, 'm').asMilliseconds() }),
+  timeout: schema.oneOf([schema.number(), schema.duration()], { defaultValue: 120000 }),
```

Comment:
The default value has to be a value that satisfies both schemas, so it must be a number.

Reply:
That's odd, seems like it should just need to satisfy one